api/src/main/java/io/grpc/CompositeCallCredentials.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc;
+
+import com.google.common.base.Preconditions;
+import java.util.concurrent.Executor;
+
+/**
+ * Uses multiple {@code CallCredentials} as if they were one. If the first credential fails, the
```

Comment:
nit: it really supports only 2 and not "multiple".

Reply:
No I am not suggesting adding a constructor to create a tree - just that the comment can be more precise to say *two* instead of multiple. But not a big deal.